src/python/pybind11/src/pyxrt.cpp

```diff
@@ -40,27 +35,40 @@ m.attr("XCL_BO_FLAGS_NONE") = py::int_(XCL_BO_FLAGS_NONE);
 
 py::enum_<xclBOSyncDirection>(m, "xclBOSyncDirection")
     .value("XCL_BO_SYNC_BO_TO_DEVICE", xclBOSyncDirection::XCL_BO_SYNC_BO_TO_DEVICE)
-    .value("XCL_BO_SYNC_BO_FROM_DEVICE", xclBOSyncDirection::XCL_BO_SYNC_BO_FROM_DEVICE);
-
+    .value("XCL_BO_SYNC_BO_FROM_DEVICE", xclBOSyncDirection::XCL_BO_SYNC_BO_FROM_DEVICE)
+    .value("XCL_BO_SYNC_BO_GMIO_TO_AIE", xclBOSyncDirection::XCL_BO_SYNC_BO_GMIO_TO_AIE)
+    .value("XCL_BO_SYNC_BO_AIE_TO_GMIO", xclBOSyncDirection::XCL_BO_SYNC_BO_AIE_TO_GMIO);
+
+
+py::enum_<ert_cmd_state>(m, "ert_cmd_state")
+    .value("ERT_CMD_STATE_NEW", ert_cmd_state::ERT_CMD_STATE_NEW)
+    .value("ERT_CMD_STATE_QUEUED", ert_cmd_state::ERT_CMD_STATE_QUEUED)
+    .value("ERT_CMD_STATE_COMPLETED", ert_cmd_state::ERT_CMD_STATE_COMPLETED)
+    .value("ERT_CMD_STATE_ERROR", ert_cmd_state::ERT_CMD_STATE_ERROR)
+    .value("ERT_CMD_STATE_ABORT", ert_cmd_state::ERT_CMD_STATE_ABORT)
+    .value("ERT_CMD_STATE_SUBMITTED", ert_cmd_state::ERT_CMD_STATE_SUBMITTED)
+    .value("ERT_CMD_STATE_TIMEOUT", ert_cmd_state::ERT_CMD_STATE_TIMEOUT)
+    .value("ERT_CMD_STATE_NORESPONSE", ert_cmd_state::ERT_CMD_STATE_NORESPONSE)
+    .value("ERT_CMD_STATE_SKERROR", ert_cmd_state::ERT_CMD_STATE_SKERROR)
+    .value("ERT_CMD_STATE_SKCRASHED", ert_cmd_state::ERT_CMD_STATE_SKCRASHED)
+    .value("ERT_CMD_STATE_MAX", ert_cmd_state::ERT_CMD_STATE_MAX);
 
 /*
  *
  * XRT:: UUID (needed since UUID classes passed outside of objects)
  *
  */
 py::class_<xrt::uuid>(m, "uuid")
-   .def(py::init<char *>())
-   .def("get",
-      [](xrt::uuid & u){
-        py::array_t<unsigned char> result = py::array_t<unsigned char>(16);
-        py::buffer_info bufinfo = result.request();
-        unsigned char* bufptr = (unsigned char*) bufinfo.ptr;
-        memcpy(bufptr,u.get(),16);
-        return result;
-     }
-    )
-    .def("to_string", &xrt::uuid::to_string)
-    ;
+    .def(py::init<char *>())
+    .def("get", [](xrt::uuid & u) {
+                    py::array_t<unsigned char> result = py::array_t<unsigned char>(16);
+                    py::buffer_info bufinfo = result.request();
+                    unsigned char* bufptr = (unsigned char*) bufinfo.ptr;
+                    memcpy(bufptr,u.get(),16);
+                    return result;
+                }
+        )
```

Comment:
Can we get rid of this get() function?  All APIs were changed to change xrt::uuid references so getting the underlying native uuid_t should not not be necessary.

Reply:
Done